openc3-init/plugins/packages/openc3-tool-admin/src/tools/OpenC3Admin/tabs/PluginsTab.vue

```diff
@@ -342,11 +343,14 @@ export default {
           okText: 'Delete',
           cancelText: 'Cancel',
         })
-        .then(function (dialog) {
+        .then((dialog) => {
+          this.alert = `Removing plugin ${plugin} ...`
+          this.alertType = 'success'
+          this.showAlert = true
```

Comment:
Removing a plugin can take a while so I wanted to notify the user that it started